rebar.config

```diff
@@ -107,7 +105,9 @@
         {runtime_tools, load},
         {tools, load},
         {mcd, load},
-        {vmq_mqtt5_demo_plugin, load}
+        {vmq_mqtt5_demo_plugin, load},
+        {jwerl, load},
```

Comment:
Why are we loading jwerl & shackle here? These are used only in events sidecar plugin AFAIK.

Reply:
Since we are not loading external plugins in the release config. so the library inside the plugin's rebar configs does not load. So I am explicitly loading them here so they will be included in the release build.